src/pages/en/guides/deploy/github.md

```diff
@@ -14,16 +14,36 @@ You can deploy an Astro site to GitHub Pages by using [GitHub Actions](https://g
 Astro maintains the official `withastro/action` to deploy your project with very little configuration. Follow the instructions below to deploy your Astro site to GitHub pages, and see [the package README](https://github.com/withastro/action) if you need more information.
 
 1. Set the [`site`](/en/reference/configuration-reference/#site) and, if needed, [`base`](/en/reference/configuration-reference/#base) options in `astro.config.mjs`.
-    - `site` should be something like `https://<YOUR_USERNAME>.github.io`
-    - `base` should be your repository’s name starting with a forward slash, for example `/my-repo`.
+    
+    ```js title="astro.config.mjs"
```

Comment:
highlight base and site options
```suggestion
    ```js title="astro.config.mjs" ins={4-5}
```

Reply:
I debated this myself, but sure, let's do it!